package.json

```diff
@@ -2,7 +2,7 @@
   "license": "MIT",
   "scripts": {
     "test": "npm run eslint && npm run abaplint",
-    "merge": "abapmerge -f src/zabapgit.prog.abap > zabapgit.abap",
+    "merge": "abapmerge -f src/zabapgit.prog.abap > zabapgit.abap -c zabapgit_full",
```

Comment:
did you mean `abapmerge -f src/zabapgit.prog.abap -c zabapgit_full > zabapgit.abap` ?

Reply:
Yes, you're right.

Reply:
I fixed.